src/components/editor-page/render-context/iframe-renderer-to-editor-communicator-context-provider.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import React, { useContext, useEffect, useMemo } from 'react'
+import { IframeRendererToEditorCommunicator } from '../../render-page/iframe-renderer-to-editor-communicator'
+import { useSelector } from 'react-redux'
+import { ApplicationState } from '../../../redux'
+
+const IFrameRendererToEditorCommunicatorContext =
+  React.createContext<IframeRendererToEditorCommunicator | undefined>(undefined)
```

Comment:
Why do you use `React.` instead of a direct import?

Reply:
no specific reason